OpenSim/Simulation/Model/Smith2016Ligament.cpp

```diff
@@ -0,0 +1,377 @@
+/* ---------------------------------------------------------------------------- *
+ *                             Smith2016Ligament.cpp                            *
+ * ---------------------------------------------------------------------------- *
+
+ * ---------------------------------------------------------------------------- */
+
+#include <OpenSim/Simulation/Model/Model.h>
+#include <OpenSim/Simulation/Model/PointForceDirection.h>
+#include "Smith2016Ligament.h"
+
+using namespace SimTK;
+using namespace OpenSim;
+
+
+//=============================================================================
+// CONSTRUCTOR(S) AND DESTRUCTOR
+//=============================================================================
+//_____________________________________________________________________________
+/**
+ * Default constructor
+ */
+Smith2016Ligament::Smith2016Ligament() : Force()
+{
+    constructProperties();
+	setNull();
+}
```

Comment:
setNull() usually calls setAuthors(), setPublications() so they show values other than default in Object browser of GUI.